docs/source/cli.rst

```diff
@@ -1,72 +1,270 @@
-                            Prior indices by rownumber (0 is first rownumber).
-      --included_dataset [INCLUDED_DATASET [INCLUDED_DATASET ...]]
-                            A dataset with papers that should be includedCan be used multiple times.
-      --excluded_dataset [EXCLUDED_DATASET [EXCLUDED_DATASET ...]]
-                            A dataset with papers that should be excludedCan be used multiple times.
-      --prior_dataset [PRIOR_DATASET [PRIOR_DATASET ...]]
-                            A dataset with papers from prior studies.
-      --state_file STATE_FILE, -s STATE_FILE, --log_file STATE_FILE, -l STATE_FILE
-                            Location to store the state of the simulation.
-      -m MODEL, --model MODEL
-                            The prediction model for Active Learning. Default: 'nb'.
-      -q QUERY_STRATEGY, --query_strategy QUERY_STRATEGY
-                            The query strategy for Active Learning. Default: 'max'.
-      -b BALANCE_STRATEGY, --balance_strategy BALANCE_STRATEGY
-                            Data rebalancing strategy mainly for RNN methods. Helps against imbalanced dataset with few inclusions and many exclusions. Default: 'double'
-      -e FEATURE_EXTRACTION, --feature_extraction FEATURE_EXTRACTION
-                            Feature extraction method. Some combinations of feature extraction method and prediction model are impossible/ill advised.Default: 'tfidf'
-      --init_seed INIT_SEED
-                            Seed for setting the prior indices if the --prior_idx option is not used. If the option --prior_idx is used with one or more index, this option is ignored.
-      --n_instances N_INSTANCES
-                            Number of papers queried each query.Default 1.
-      --n_queries N_QUERIES
-                            The number of queries. By default, the program stops after all documents are reviewed or is interrupted by the user.
-      -n N_PAPERS, --n_papers N_PAPERS
-                            The number of papers to be reviewed. By default, the program stops after all documents are reviewed or is interrupted by the user.
-      --abstract_only       Simulate using the labels of abstract screening. This is option is useful if there is both a column for abstract and final screening available in the dataset. Default False.
-      --verbose VERBOSE, -v VERBOSE
-                            Verbosity
+.. program:: asreview simulate
+
+.. option:: dataset
+
+    A dataset to simulate
+
+.. option:: -m, --model MODEL
+
+    The prediction model for Active Learning. Default: :code:`nb`. (See available
+    options below: `Classifiers`_)
+
+.. option:: -q, --query_strategy QUERY_STRATEGY
+
+    The query strategy for Active Learning. Default: :code:`max`. (See
+    available options below: `Query strategies`_)
+
+.. option:: -b, --balance_strategy BALANCE_STRATEGY
+
+    Data rebalancing strategy mainly for RNN methods. Helps against imbalanced
```

Comment:
What is RNN?

Reply:
Recurrent Neural Networks (like most LSTM-based models).

Reply:
I'm not sure on this sentence. @qubixes this holds for all models, isn't it?

Reply:
It seemed to have the most effect for Naive Bayes.

Reply:
So remove this line?

Reply:
For now, I decided to remove the line.